### PR TITLE
Update UML Diagrams and formatting for AddCommand

### PR DESCRIPTION
**Pull request by jeffreytjs**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1448,15 +1458,19 @@ testers are expected to do more *exploratory* testing.
...
    1. Test case: Minimal information e.g. `add ct/w n/John Ptd Ltd p/98766789 addr/John street, block 123
       , #01-01`<br>
-      Expected: Adds a warehouse with the above details to the list and displayed on the GUI.
-   1. Test case: With remarks e.g. `add ct/w n/John Ptd Ltd p/98766789 addr/John street, block 123, #01-01
-      r/Largest warehouse`<br>
-      Expected: Adds the warehouse to the list, including the remark
+      Expected: Adds a warehouse with the above details to the warehouse list and is displayed on the GUI.
+   1. Test case: Duplicate warehouse by name e.g. `add ct/w n/John Ptd Ltd p/99999999 addr/John 2 street
+      , block 222, #02-02`<br>
+      Expected: No warehouse is added. Adds the warehouse to the list, including the remark
```
Comment (zhengweii) [inline docs/DeveloperGuide.md]: Is there a mistake here? "No warehouse is added. Adds the warehouse to the list"

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1448,15 +1458,19 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: Duplicate name With remarks e.g. `add ct/w n/John Ptd Ltd p/98766789 addr/John street, block
+      123, #01-01
+        r/Largest warehouse`<br>
+        Expected: Adds the warehouse to the list, including the remark
```
Comment (zhengweii) [inline docs/DeveloperGuide.md]: Ohh does this test case add the warehouse to the list as well??

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jeffreytjs) [reply docs/DeveloperGuide.md]: Yup, i was in the midst of editing both and got distacted by my UML hahahaa

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (jeffreytjs) [reply docs/DeveloperGuide.md]: I've resolved this as well as standardise it with adding supplier! Good catch!

**Review by zhengweii — APPROVED** [review]
LGTM

**Pull request merged**